packages/paste-theme/__tests__/themes.ts

```diff
@@ -0,0 +1,17 @@
+import {ConsoleTheme, DefaultTheme, SendGridTheme} from '../src';
+
+describe('Default theme', () => {
+  it('should match the snapshot', () => {
+    expect(DefaultTheme).toMatchSnapshot();
+  });
+});
+describe('Console theme', () => {
+  it('should match the snapshot', () => {
+    expect(ConsoleTheme).toMatchSnapshot();
+  });
+});
+describe('Sendgrid theme', () => {
+  it('should match the snapshot', () => {
+    expect(SendGridTheme).toMatchSnapshot();
+  });
+});
```

Comment:
These will obviously change as we add tokens or change token values, but I think it's worth it to have the fallback of knowning what the generated theme shape is, when / if we come to refactor the theming

Reply:
What if we did 
`expect(Object.keys(SendGridTheme).sort()).toMatchSnapshot()`?

Reply:
Yeah, I buy that. We're only testing the buckets are right, don't care about the tokens, good shout